pyproject.toml

```diff
@@ -60,6 +60,7 @@ mkdocs-material-extensions = "^1.1.1"
 mkdocs-glightbox = "^0.3.1"
 mike = "^1.1.2"
 typer-cli = "^0.0.13"
+mkdocs-codeinclude-plugin = "^0.2.1"
```

Comment:
Not used?

Reply:
I ended up not using it at all. Should I remove it?

It would be a nice tool further in time. How can we draw attention to it?

Reply:
If it is not used, It should be removed. You mentioned that you are using this [plugin](https://facelessuser.github.io/pymdown-extensions/extensions/snippets/). What are the differences between them? Is one better than the other one?
If it is so and we need to change it, it makes sense to create an issue. This is how you draw attention to it. 😄

Reply:
I cannot say whether one is better, but I think the one we are using has more features currently. The thing that [snippets](https://facelessuser.github.io/pymdown-extensions/extensions/snippets/) is missing is using [expressions](https://github.com/rnorth/mkdocs-codeinclude-plugin#usage) to find and include code from other files.

[Codeinclude](https://github.com/rnorth/mkdocs-codeinclude-plugin), however, cannot import files locally or from the cloud via URL.

That's why I think we can use the 2 plugins concurrently.

NOTE: I couldn't make [codeinclude](https://github.com/rnorth/mkdocs-codeinclude-plugin) work for some reason, so even if the problem is in me, it's a clear indicator that its documentation isn't great currently.

Reply:
Okay interesting findings. I would still remove codeinclude and create an issue about why and where we can use this plugin. And then tackle it in another PR.

Reply:
https://github.com/bakdata/kpops/issues/179